src/App.jsx

```diff
@@ -2,17 +2,16 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
 
   const addOne = () => {
-    // write code here
+    setCount(curentCount => curentCount + 1);
```

Comment:
There is a typo in the parameter name. It should be 'currentCount', not 'curentCount'.